dune/stuff/common/vector.hh

```diff
@@ -144,9 +144,9 @@ struct VectorAbstraction< Dune::Stuff::Common::FieldVector< K, SIZE > >
 
   static const  size_t static_size = SIZE;
 
-  static inline VectorType create(const size_t sz)
+  static inline VectorType create(const size_t /*sz*/)
   {
-    return VectorType(sz);
+    return VectorType();
   }
```

Comment:
I think there was a compiler warning/error, but I dont know what it was exactly anymore. Maybe I did the wrong thing to fix it, so I will revert it for now and see if it occurs again.